console/src/main/webapp/manager/app/components/delegations/delegations.es6

```diff
@@ -0,0 +1,18 @@
+import 'components/delegations/delegations.tpl'
+import 'services/delegations'
+
+class DelegationsController {
+  static $inject = [ 'Delegations' ]
+
+  constructor (Delegations) {
+    // this.delegations = Delegations.query()
```

Comment:
Do we need to keep this comment ?

Reply:
Yep, this will replace the following line when @Vampouille will address [this comment](https://github.com/georchestra/georchestra/pull/1969#issuecomment-376935677). :sunglasses: